packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -2183,6 +2184,115 @@ curl -v -X POST \
 }' "https://${yourOktaDomain}/api/v1/authn/factors/clf198rKSEWOSKRIVIFT/lifecycle/resend"
 ```
 
+#### Enroll Okta Email Factor
+
+
+Enrolls a user with the Okta `email` factor using the users primary email address.  An email message with an OTP is sent to the user during enrollment and must be [activated](#activate-call-factor) by following the `next` link relation to complete the enrollment process.
```

Comment:
user's

Reply:
Also if you're talking about the Factor object in Okta, it should be capitalized. Applies here and throughout. 

Rest of page was fixed in this PR: https://github.com/okta/okta-developer-docs/pull/903